### DISCUSSION TITLE
Putting up collateral to be part of a DAO / SubDAO

### DISCUSSION THREAD

**Discussion by JakeHartnell** [DA0-DA0/dao-contracts#531] [Ideas]
*User story:* As a community seeking to protect itself, a DAO wants to require members of a SubDAO put up collateral to be a part of it. Should they be removed due to bad behavior, their collateral would be slashed. Thus, SubDAO members are incentivized to perform their job ethically.

This idea was brought up during a recent Twitter spaces and I do think it's very interesting. Collateral could also be staked to further align incentives.

There a few ways that this could be implemented. Wanted to capture this idea and discuss:
- Is this a useful thing?
- How might we implement it?
- How might DAO members eventually get their deposit back?

**Comment by MetaHousingSX8** [comment]
There comes a point in the rabbit hole where we’re all so deep that there’s intrinsic trust and knowing between all parties involved, in my opinion.
